src/uiScreens.c

```diff
@@ -718,7 +718,9 @@ void ui_displayPoolRelayScreen(
 	char firstLine[20];
 	explicit_bzero(firstLine, SIZEOF(firstLine));
 	{
+		#ifndef FUZZING
 		STATIC_ASSERT(sizeof(relayIndex + 1) <= sizeof(unsigned), "oversized type for %u");
```

Comment:
What's the problem here?

Reply:
here `relayIndex` is a `size_t`, which is 64 bits on x64 which one should preferably use for fuzzing, but `unsigned` is 32 bits.
I am not sure what was the intent behind this check originally however